src/pages/LoginPage/LoginPage.jsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import "./LoginPage.css";
+import { useState } from "react";
+import axios from "axios";
+import Nav from '../../components/Nav/Nav'
+import { Link, useNavigate } from "react-router-dom";
+import { useAuth } from "../../context/context"
+
+const LoginPage = () => {
+  const [email, setEmail] = useState("");
+  const [pass, setPass] = useState("");
+
+  const { dispatch } = useAuth();
+  const navigate = useNavigate();
+
+  const loginClickHandler = async (e) => { 
+    const password = pass;
+    e.preventDefault();
+    try{
+      const resp = await axios.post("/api/auth/login",
+      { email,
+        password})
+      localStorage.setItem("token",resp.data.encodedToken)
+      localStorage.setItem("user",JSON.stringify(resp.data.foundUser))
+      dispatch({ type: "SUCCESS", payload:resp.data.encodedToken})
+      navigate("/videos")
+      console.log(resp)
+    } catch(err){
+      alert(`Enter Correct Email & Password : ${err}`)
+
+    }
+  }
+
+  const loginDummyClickHandler = async (e) => { 
+    e.preventDefault();
+    try{
+      const resp = await axios.post("/api/auth/login",
+      { email: "adarshbalika@gmail.com",
+        password: "adarshBalika123"})
+      localStorage.setItem("token",resp.data.encodedToken)
```

Comment:
You are only storing the token. I just want to know are you using it anywhere?

Reply:
No... I am not using it anywhere